common/pom.xml

```diff
@@ -145,5 +118,16 @@
               </plugins>
           </build>
       </profile>
+      <profile>
+        <id>setPropFileName</id>
+        <activation>
+          <property>
+            <name>!propertiesFileName</name>
+          </property>
+        </activation>
+        <properties>
+          <propertiesFileName>jbossts-properties.xml</propertiesFileName>
+        </properties>
+      </profile>
```

Comment:
I do not see why you need a profile. I guess you should be fine defining a property:
```
<properties>
  <propertiesFileName>jbossts-properties.xml</propertiesFileName>
</properties>
```

Reply:
Okay, I will provide separate property for this

Reply:
Thanks